File: php/test/Maths/ComparisonTest.php
<?php

namespace Armakuni\SimpleMutationTestingKata\Maths;

use PHPUnit\Framework\TestCase;
use function PHPUnit\Framework\assertFalse;
use function PHPUnit\Framework\assertTrue;

class ComparisonTest extends TestCase
{
    public function test1IsEqualTo1()
    {
        $this->assertTrue(Comparison::isEqual(1,1));
    }

    public function test2IsNotEqualTo1()
    {
        $this->assertFalse(Comparison::isEqual(2,1));
    }

    public function test1IsLessThan2()
    {
        $this->assertTrue(Comparison::lessThan(1,2));
    }

    public function test2IsNotLessThan1()
    {
        $this->assertFalse(Comparison::lessThan(2,1));
    }

    public function test2IsGreaterThan1()
    {
        $this->assertTrue(Comparison::greaterThan(2,1));
    }

    public function test1IsNotGreaterThan2()
    {
        $this->assertFalse(Comparison::greaterThan(1,2));
    }
}

File: java/src/test/java/com/armakuni/maths/ComparisonTest.java
package com.armakuni.maths;

import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.*;
class ComparisonTest {

    @Test
    void that1IsEqualTo1() {
        Comparison c = new Comparison();
        assertTrue(c.isEqual(1, 1));
    }

    @Test
    void that2IsNotEqualTo1() {
        Comparison c = new Comparison();
        assertFalse(c.isEqual(2, 1));
    }

    @Test
    void that1IsLessThan2() {
        Comparison c = new Comparison();
        assertTrue(c.lessThan(1, 2));
    }

    @Test
    void that2IsNotLessThan1() {
        Comparison c = new Comparison();
        assertFalse(c.lessThan(2, 1));
    }

    @Test
    void that2IsGreaterThan1() {
        Comparison c = new Comparison();
        assertTrue(c.greaterThan(2, 1));
    }

    @Test
    void that1IsNotGreaterThan2() {
        Comparison c = new Comparison();
        assertFalse(c.greaterThan(1, 2));
    }
}